services/graphService.js

```diff
@@ -28,19 +28,45 @@ GraphService.prototype.findByPath = function(path) {
 	})
 }
 
+/**
+ * @param paging
+ * @return {list, meta, owners}
+ * @private
+ */
+GraphService.prototype._listWithOwners = function(filter, sort, select, paging) {
+	var that = this
+	var listData=[]
+	return this.countAndFind(filter, sort, select, paging)
+		.then((data) => {
+			listData = data
+			var creatorIds = {}
+			// only interested in unique ids
+			for (var row of data.list) {
+				if (row._creator)
+					creatorIds[row._creator] = true
+			}
+			return that.getOwnersInfo(Object.keys(creatorIds))
```

Comment:
@kschzt but creatorIds are not unique if an array, which is why i had them into a {} so that keys overwrite each other